app.js

```diff
@@ -1 +1,32 @@
+'use strict';
+//var firstPikeMain ={
+var hours = ['6a','7a','8a','9a','10a','11a','12p','1p','2p','3p','4p','5p','6p','7p','8p'];
+var firstPike = {
+  minCust: 23, //object properties //for mathrandom
+  maxCust: 65, //object ptoperties//for mathrandom
+};
 
+var avgCust = function() {
+  var a = Math.floor(Math.random() * (firstPike.maxCust - firstPike.minCust)) + firstPike.minCust;
```

Comment:
Gotta use better variable names than "a". We've talked extensively about descriptive variables. If you're using a variable like this because it's kind of a throwaway for this function, do `return Math.floor(Math.random() * (firstPike.maxCust - firstPike.minCust)) + firstPike.minCust;`